humble/distribution.yaml

```diff
@@ -4553,6 +4553,24 @@ repositories:
       url: https://gitlab.com/ApexAI/point_cloud_msg_wrapper
       version: rolling
     status: developed
+  point_cloud_transport:
+    doc:
+      type: git
+      url: https://github.com/ros-perception/point_cloud_transport_tutorial.git
+      version: rolling
```

Comment:
I'm just noticing that the branch here is `rolling`, which is the same as in [rolling/distribution.yaml](https://github.com/ros/rosdistro/blob/676cfdb79ca36a2cc5d9745a48e8e8b0768a9765/rolling/distribution.yaml#L3731-L3749).  However, the released version is 1.0.12 here, but 2.0.0 in Rolling.  Is this all correct?

Reply:
For the documentation we only have `rolling`

Reply:
Using a different repository for the documentation is unusual.  It's not disallowed, but what is listed here in `doc` is what we will use when generating API documentation for the package.  So what you are specifying here is that we should generate API documentation for `point_cloud_transport_tutorial`, not for `point_cloud_transport`.

My suggestion here is that you change this to https://github.com/ros-perception/point_cloud_transport.git; the documentation for the tutorial will be specified via the `point_cloud_transport_tutorial` key below.  This should probably be fixed in rolling/distribution.yaml as well.

Further, the source branch below also points to `rolling`, which I think should be `humble` instead.